docs/BUILD.md

```diff
@@ -25,7 +25,7 @@ projects
 
 ## Build Process
 
-The build scripts in this repository have been designed to work with Node.js v17 or later.
+The build scripts in this repository have been designed to work with Node.js v16 or later.
```

Comment:
Because of the `.nvmrc`

Reply:
@steve-haar had some comments comments about the version of Node we need to use. I didn't understand if the requirement was updated in the `.nvmrc` too. Would be good to have some comment from him too.

Reply:
The project has been set up to work with node 16. This is just a mistake in the documentation. Thanks for finding it.